src/main.rs

```diff
@@ -67,7 +72,17 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    let mut rabbitspopulation = starting_rabbits;
+    let mut count = 0;
+    while rabbitspopulation != 1 {
+        count += 1;
+        if rabbitspopulation % 2 == 0 {
+            rabbitspopulation = rabbitspopulation / 2;
```

Comment:
One thing I saw from some other students (that I didn't think to do myself) was to do `rabbitspopulation /= 2;` on this line. Kind of neat!

Also, Rust uses `snake_case`, so it might make more sense to name the variable `rabbits_population`